xcresult/model.py

```diff
@@ -588,6 +588,20 @@ class ActionTestSummaryIdentifiableObject(ActionAbstractTestSummary):
 
     identifier: Optional[str]
 
+class ActionTestRepetitionPolicySummary(XcresultObject):
+    """Generated from xcresulttool format description.
+
+    - ActivityLogAnalyzerStep
+      * Kind: object
+      * Properties:
+        + iteration: Optional[int]
```

Comment:
This looks like it was generated by hand. This is an auto-generated file (I know, there's no documentation so that's on me). To generate it, cd into `generator` and run `python __init__.py`

Reply:
Just took a look. This and a few other things come in with Xcode 12.5. I've changed the versioning to make this "released" (previous version was 0.5.0 and it is now 12.4.0 to match Xcode). This should be generated with the version bumped to 12.5.0.

Reply:
This assumes there is some venv setup somewhere. What does the env here expect?

```
  File "__init__.py", line 18
    def get_indentation(line: str) -> int:
                            ^
SyntaxError: invalid syntax
```

Reply:
Python 3 :) 

You'll also need poetry. There's no prescribed way for handling this. It's entirely user dependent.

Reply:
This has been merged and published as version `12.5.0`